### PR TITLE
[POM-13] initial accommodation search

### PR DESCRIPTION
**Pull request by sfc-gh-krzymkowski**

### CODE REVIEW

File: src/app/find-help/search-result/search-result.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  @Input()
+  location?: OfferLocation;
+  @Input()
+  title!: string;
+  @Input()
+  description!: string;
+  @Input()
+  posted?: Date;
+  @Input()
+  attributes?: {
```
Comment (Vodzoo) [inline src/app/find-help/search-result/search-result.component.ts:22]: Create interfece.

File: src/app/find-help/accommodation-search/accommodation-search.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+})
+export class AccommodationSearchComponent {
+  results: AccommodationOffer[] = [];
+  total: number = NaN;
+  loading = false;
+  constructor(private http: HttpClient) {}
+
+  search(query: AccommodationQuery) {
+    this.loading = true;
+    this.http
```
Comment (Vodzoo) [inline src/app/find-help/accommodation-search/accommodation-search.component.ts:29]: We should generate services from the swagger file.

Comment (sfc-gh-krzymkowski) [reply src/app/find-help/accommodation-search/accommodation-search.component.ts:29]: I suggest setting that up in another PR

File: src/app/api/types.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export type OfferLocation = { city: string; region: string };
```
Comment (Vodzoo) [inline src/app/api/types.ts:1]: Why type and not interface?

**Review by Vodzoo — APPROVED** [review]

**Pull request merged**